config/activity_labels/medical/r19.yaml

```diff
@@ -0,0 +1,28 @@
+version: "1"
+title: "Needle_Chest_Decomp"
+labels:
+  # Item:
+  # - id: Integer identification number of the label.
+  # - label: Short human-readable semantic string to succinctly represent the
+  #          label.
+  - id: 0
+    label: "background"
+    full_str: "background"
+  - id: 1
+    label: "locate-insertion-site"
```

Comment:
This sounds to me like a step worth leaving out of training... unless the user tends to place their hand a the midclavicle or something. We'll see!